packages/app/src/overlay/canvas/ValidatorMetrics/index.tsx

```diff
@@ -62,13 +62,19 @@ export const ValidatorMetrics = () => {
   let validatorOwnStake = new BigNumber(0)
   let otherStake = new BigNumber(0)
   if (validatorInEra) {
-    const { others, own } = validatorInEra
-    others.forEach(({ value }) => {
-      otherStake = otherStake.plus(value)
-    })
+    const { own, total } = validatorInEra
+
+    // Set validator own stake
     if (own) {
       validatorOwnStake = new BigNumber(own)
     }
+
+    // Calculate nominator stake as total minus own stake
+    // This ensures we get the correct total even if we're missing some nominator data
+    if (total) {
+      const totalStake = new BigNumber(total)
+      otherStake = totalStake.minus(validatorOwnStake)
```

Comment:
This should never happen with valid data, but just to be defensive, ensure that number never goes below zero.

```suggestion
      otherStake = BigNumber.max(0, totalStake.minus(validatorOwnStake))
```